jamjar/jamjar/search/views.py

```diff
@@ -0,0 +1,207 @@
+                "256": "https://s3.amazonaws.com/jamjar-videos/dev/46324b43-01f3-460c-8da9-dab840b1153b/thumb-256.jpg",
+                "512": "https://s3.amazonaws.com/jamjar-videos/dev/46324b43-01f3-460c-8da9-dab840b1153b/thumb-512.jpg",
+                "1024": "https://s3.amazonaws.com/jamjar-videos/dev/46324b43-01f3-460c-8da9-dab840b1153b/thumb-1024.jpg"
+              },
+              "concert": 2,
+              "user": {
+                "id": 2,
+                "username": "drew",
+                ...
+              },
+            },
+            ...
+          ],
+          "artist": [
+            {
+              "id": 2,
+              "name": "OutKast",
+              "spotify_id": "1G9G7WwrXka3Z1r7aIDjI7",
+              "genres": [
+                "hip hop"
+              ],
+              "images": [
+                {
+                  "url": "https://i.scdn.co/image/cf78e460a538b48dec7ef55064cdc76042511a07",
+                  "height": 809,
+                  "width": 1000
+                },
+                ...
+              ],
+              "unofficial": false
+            }
+          ],
+          "venue": [
+            {
+              "id": 1,
+              "name": "World Cafe Live",
+              "place_id": "ChIJ7RhuX0_GxokRjL5qFrDINys",
+              "unofficial": false,
+              "formatted_address": "3025 Walnut St, Philadelphia, PA 19104, United States",
+              "lat": "39.95219830",
+              "lng": "-75.18517170",
+              "utc_offset": -300,
+              "website": "http://philly.worldcafelive.com/",
+              "city": "Philadelphia",
+              "state": "Pennsylvania",
+              "state_short": "PA",
+              "country": "United States",
+              "country_short": "US"
+            }
+          ],
+          "concert": [
+            {
+              "id": 1,
+              "date": "2016-02-28",
+              "venue": {
+                "id": 1,
+                "name": "World Cafe Live",
+                "place_id": "ChIJ7RhuX0_GxokRjL5qFrDINys",
+                "unofficial": false,
+                "formatted_address": "3025 Walnut St, Philadelphia, PA 19104, United States",
+                "lat": "39.95219830",
+                "lng": "-75.18517170",
+                "utc_offset": -300,
+                "website": "http://philly.worldcafelive.com/",
+                "city": "Philadelphia",
+                "state": "Pennsylvania",
+                "state_short": "PA",
+                "country": "United States",
+                "country_short": "US"
+              },
+              "videos": [ ... ]
+              "artists": [ .. ]
+            },
+            ...
+          ],
+          "user": []
+        } 
+    """
+
+    @authenticate
+    def get(self, request):
+        query_string = request.GET.get("q", "").strip()
+
+        if not query_string:
+            return self.error_response({"error": "Search query is empty"}, 500)
```

Comment:
You can just pass in a string here, `error_response` will put it into an "error"
https://github.com/projectjamjar/masonjar/blob/master/jamjar/jamjar/base/models.py#L24